AudioSystem/AudioConfigSequence.cs

```diff
@@ -91,12 +91,15 @@ public void SetParameter(string key, int value)
 			clipIndexParameters[key] = value;
 		}
 
-		public int GetClipIndex(AudioConfig audioConfig)
+		public int GetClipIndex(int audioConfigIndex)
```

Comment:
This function name no longer makes sense to me. GetIndex but you pass another index as a parameter?

Reply:
It gets the index of the audioClip which should play for a given audioConfig (by index). You're asking for the correct (or a random) clip of a certain config in the sequence.